src/modules/navigation/sections/Navigation.vue

```diff
@@ -62,19 +52,32 @@
 						</NcCounterBubble>
 					</template>
 
-					<NavigationTableItem v-for="table in getArchivedTables"
-						:key="table.id"
-						:filter-string="filterString"
-						:table="table" />
+					<NavigationTableItem v-for="table in getArchivedTables" :key="table.id"
+						:filter-string="filterString" :table="table" />
 				</NcAppNavigationItem>
 			</ul>
+			<div v-if="contextsLoading" class="icon-loading" />
+			<ul v-if="!contextsLoading">
+				<NcAppNavigationCaption :name="t('tables', 'Contexts')">
```

Comment:
User facing naming would be "Applications"
```suggestion
				<NcAppNavigationCaption :name="t('tables', 'Applications')">
```